client/lib/domains/constants.js

```diff
@@ -17,7 +17,40 @@ const registrar = {
 	MAINTENANCE: 'Registrar TLD Maintenance'
 };
 
+const tlds = {
```

Comment:
IMO that is going to be a pain to maintain, when someone needs to add a new tld,
he also needs to remember to put it here...

What you say about using `QueryProductsList` and then a selector something like:
```js
fetch( 'https://public-api.wordpress.com/rest/v1/products' )
.then( response => response.json() )
.then( productsMap => {
	const tlds = {};

	Object.keys( productsMap )
	.map( key => productsMap[ key ] )
	.filter( product => product.is_domain_registration )
	.map( product => ( {
		tld: product.product_name.indexOf( '.' ) > -1 ? product.product_name.match( /\.([^ ]+)/ )[ 1 ] : 'DEFAULT',
		product_slug: product.product_slug
	} ) )
	.forEach( product => tlds[ product.tld ] = product.product_slug )
	
	tlds[ 'com' ] = tlds[ 'net' ] = tlds[ 'org' ] = tlds.DEFAULT;

	delete tlds.DEFAULT;

	return tlds;
} )
.then( tlds => {
	console.log( tlds );	
} )
```